backend/hct_mis_api/config/settings.py

```diff
@@ -148,17 +147,12 @@
         "sslrootcert": "/code/psql-cert.crt",
         "options": "-c search_path=erp",
     }
-    DATABASES["registration_datahub"]["OPTIONS"] = {
-        "sslmode": "verify-full",
-        "sslrootcert": "/code/psql-cert.crt",
-    }
 
 # If app is not specified here it will use default db
 DATABASE_APPS_MAPPING: Dict[str, str] = {
     "cash_assist_datahub": "cash_assist_datahub_ca",
     "mis_datahub": "cash_assist_datahub_mis",
     "erp_datahub": "cash_assist_datahub_erp",
-    "registration_datahub": "registration_datahub",
```

Comment:
this one too, without this line all migration from registration_datahub will be applied on the default database, but should be on registration_datahub